index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <a href="#"><h1>My Logo</h1></a>
+        <a href="#">
+            <img src="images/Union.png" alt="Union Image">
+        </a>
+    </nav>
+
+    <main>
+        <section class="header">
+            <h1>I’m Mariane Glad to see you!</h1>
+            <p>
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="#">LET’S CONNECT</a>
+            <div class="social">
+                <a href="#">
+                    <img src="images/Twitter.svg" alt="Twitter icon">
+                </a>
+                <a href="#">
+                    <img src="images/Linkedin.svg" alt="Linkedin icon">
+                </a>
+                <a href="#">
+                    <img src="images/Medium.svg" alt="Medium icon">
+                </a>
+                <a href="#">
+                    <img src="images/Github.svg" alt="Github icon">
+                </a>
+                <a href="#">
+                    <img src="images/Angelist.svg" alt="Angelist icon">
+                </a>
+            </div>
```

Comment:
- [ ] Great job on implementing all the Icons 💪 According to the requirements however all the Icons should be grouped into an unordered list `<ul>` and the individual icons should be wrapped in a list element `<li>`